packages/gatsby-theme-store/src/components/Auth/Providers/ExternalProvider/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-extra-semi */
```

Comment:
Why is that needed?

Reply:
It was complaining about the `;(async` on line 36, but if I remove it complains about having to add a ; 🤔  And in both occasions it doesn't let me commit while I don't fix it